spec/fixtures/parameterizing_rules/user_defined/with_action_and_named_references.y

```diff
@@ -16,7 +16,7 @@ static int yyerror(YYLTYPE *loc, const char *str);
 %token <i> number
 %token <s> string
 
-%rule pair(X, Y): X ',' Y { printf("(%d, %d)\n", $X, $3); }
+%rule pair(X, Y): X ',' Y[alias] { printf("(%d, %d)\n", $X, $alias); }
```

Comment:
The [intention](https://github.com/ruby/lrama/pull/410#issuecomment-2093981195) makes sense for me. Could you update the test case grammar to match with the intention?

Reply:
Indeed it is. I updated this PR.